coralproject/talk
# [next] Implement "Read More of this Conversation"

## What does this PR do?
- https://www.pivotaltracker.com/story/show/160729882

## How do I test this PR?

- Write comments until max depth levels has been exceeded
- Reload and see a functional `Read More of this Conversation` button
